Distribution/Dockerfile

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+FROM centos:6.6 as builder
```

Comment:
Please explain the need for this (each new Docker means that we have to maintain that Dynawo is compiling on this distribution, so we should maintain as few as possible)

Reply:
This image wil publish a Dynawo distribution on Github at each release. The need for centos6.6 is to have a very olf compiler that will provide binaries that will be compatible with all new OS.